spec/suma/commerce/cart_spec.rb

```diff
@@ -219,4 +219,59 @@ def create_fake_order(cart)
       expect(described_class[item.id]).to have_attributes(offering_product: be === op) # non-eager loading
     end
   end
+
+  describe "create_checkout" do
```

Comment:
Port over all the specs, not just a handful of them. Otherwise you'll have to do code archaeology to see what was done, when you have to maintain or edit this or the API code.

Reply:
I think I ported all specs now, the API specs left-over are specific to the commerce API. I did update the specs though with commit [fc68742](https://github.com/lithictech/suma/pull/629/commits/fc68742a59bc441744a27ac92f5ef8649ddf6618)